src/applications/personalization/profile/mocks/server.js

```diff
@@ -111,4 +113,4 @@ const responses = {
   },
 };
 
-module.exports = responses;
+module.exports = delay(responses, 2000);
```

Comment:
This would be nice if we could configure whether or not to include a delay. Maybe with an cli arg when starting the mock server, or some other method?